package.json

```diff
@@ -23,7 +23,7 @@
     "@types/estree": "^0.0.42",
     "@types/jest": "25.2.1",
     "@types/node": "12.12.35",
-    "@vue/compiler-sfc": "^3.0.2",
+    "@vue/compiler-sfc": "3.0.3",
```

Comment:
Are these an intended change?

Reply:
Yes, `<script setup>` works completely differently in 3.0.2 and 3.0.3.

So kind of a breaking change - for `<script setup>` people. Those users are on the cutting edge anyway, so they probably know enough to figure out that they need the absolute latest version.